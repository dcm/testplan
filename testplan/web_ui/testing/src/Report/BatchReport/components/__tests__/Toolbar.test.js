/** @jest-environment jsdom */
// @ts-nocheck
import React from 'react';
import { Toolbar } from '../';
import { shallow } from 'enzyme';

describe('Toolbar', () => {
  it('renders correctly', () => {
    expect(shallow(<Toolbar/>)).toMatchSnapshot();
  });
});
