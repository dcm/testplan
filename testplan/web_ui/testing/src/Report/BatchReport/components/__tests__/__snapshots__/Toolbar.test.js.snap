// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Toolbar renders correctly 1`] = `
<div>
  <TopNavbar />
  <HelpModal />
  <InfoModal />
</div>
`;
